src/osha/oira/services/country.py

```diff
@@ -0,0 +1,49 @@
+from plone import api
+from plone.restapi.services import Service
+
+
+class Tools(Service):
+    """A JSON service version of @@country-tools"""
+
+    def reply(self):
+        tree = []
+        ctv = api.content.get_view("country-tools", self.context, self.request)
+        country = self.context.getId()
+        for sector in ctv.sectors:
+            groups = []
+            sector_path = "/".join([country, sector["id"]])
+            for group in ctv.get_tools(sector["id"]):
+                group_path = "/".join([sector_path, group["id"]])
+                surveys = []
+                for survey in group["surveys"]:
+                    survey_path = "/".join([group_path, survey["id"]])
+                    surveys.append(
+                        {
+                            "id": survey["id"],
+                            "path": survey_path,
+                            "title": survey["title"],
+                            "url": survey["url"],
+                            "is_published": True if survey["published"] else False,
```

Comment:
More compact, maybe more readable, IDK...
```suggestion
                            "is_published": bool(survey["published"]),
```
I leave to you the decision to keep or discard this suggestion.

It might even be that this is sufficient:
```suggestion
                            "is_published": survey["published"],
```